BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemView.swift

```diff
@@ -55,28 +62,43 @@ struct AddEditItemView: View {
     }
 
     private var content: some View {
-        ScrollView {
-            VStack(spacing: 20) {
-                if isPolicyEnabled {
-                    InfoContainer(Localizations.personalOwnershipPolicyInEffect)
-                        .accessibilityIdentifier("PersonalOwnershipPolicyLabel")
-                }
+        ScrollViewReader { proxy in
+            ScrollView {
+                VStack(spacing: 20) {
+                    if isPolicyEnabled {
+                        InfoContainer(Localizations.personalOwnershipPolicyInEffect)
+                            .accessibilityIdentifier("PersonalOwnershipPolicyLabel")
+                    }
 
-                informationSection
-                miscellaneousSection
-                notesSection
-                customSection
-                ownershipSection
+                    informationSection
+                    miscellaneousSection
+                    notesSection
+                        .onTapGesture {
+                            proxy.scrollTo(
+                                ItemSectionID.notes,
+                                anchor: .bottom
+                            )
+                        }
+                        .onChange(of: store.state.notes) { _ in
+                            proxy.scrollTo(
+                                ItemSectionID.notes,
+                                anchor: .bottom
```

Comment:
🤔 Will this end up causing problems if you have a really long note and attempt to edit text at the top of the note? Will it scroll to the bottom instead of keeping the cursor visible at the top?

Reply:
yeah thats an edge case for sure... just tested it out. How do you think it should behave?  The `ScrollView` here is preventing SwiftUI from handling the keyboard correctly with the TextField.

Reply:
You can, once it scrolls, scroll to the top and edit.

Reply:
Ideally the view should scroll to always keep the cursor visible. But SwiftUI should be doing that for us 🙁. I'm not sure what the best option is. Is it any better if we limit the text field to a certain number of lines and let it scroll within the text view? I don't think that's ideal, but maybe it'd be better than what we have. Otherwise we could see about wrapping a `UITextView` and seeing if that's any better...

Reply:
🤔 Would it be possible to have these behaviors applied?
- If the notes field doesn't have focus and the user taps there then it scrolls to bottom as it's now
- If the notes field is focused and the cursor is in the last line (or last character) of the field then keep scrolling bottom
- If the notes field is focused and the cursor is not in the last line then do not automatically scroll and let the user do the scrolling
Maybe this could be an acceptable behavior of the control. What do you think?

Reply:
I agree, this makes total sense to me.  Working on that last bullet atm so ill get back with a new commit once i figure it out!  thanks for the insight!